stylesheets/viewer.css

```diff
@@ -1,40 +1,44 @@
-:root {
```

Comment:
just a side node on this file rename, in https://github.com/nextjournal/clerk/pull/36/files#diff-8950130d83008fae9fd570854a759d6542375816ee6c4b647cae8f1cecdd276dR170 we're assuming `viewer.css` lives in resources, but maybe that won't be needed any more after this pr gets merged?